app/code/core/Mage/Oauth/Model/Server.php

```diff
@@ -397,7 +397,7 @@ protected function _saveToken()
     {
         if (self::REQUEST_INITIATE == $this->_requestType) {
             if (self::CALLBACK_ESTABLISHED == $this->_protocolParams['oauth_callback']
-                && $this->_consumer->getCallBackUrl()
+                && $this->_consumer->getCallbackUrl()
```

Comment:
The reason is BC. We cannot do `$this->_consumer->getData('callback_url))` or `$this->_consumer->callback_url` in case `getCallbackUrl()` there is an override. Magento 1 is all about flexibility and an almost "infinite" way to customize. Magic getter and setter provides yet another way.

Reply:
If that line was ever implemented (without magic getter), it will have worked without your change anyways.

function foo()

can be called as foo() or Foo(). Its the same for PHP. If implemented.

But I see your reasoning.

Reply:
No, without the change, it cannot work:

Before PR: `getCallBackUrl()` is `getData('call_back_url')` will return null because there is no such column in table `oauth_token`.
With PR: `getCallbackUrl()`  is `getData('callback_url')`, will return the value in the corresponding column in table `oauth_token`